src/activities/Elsa.Activities.Email/Activities/SendEmail/SendEmailExtensions.cs

```diff
@@ -7,17 +8,38 @@ namespace Elsa.Activities.Email
 {
     public static class SendEmailExtensions
     {
+        #region sendner
```

Comment:
I prefer if we'd use single line comments to group these methods instead of #regions.

Reply:
Okay, it's not a problem here either. But sections that contain many more lines, this method is better. Then you can collapse the individual sections.

Reply:
I think that's debatable, but let's cross that bridge when we get there :)